.github/workflows/main.yml

```
@@ -3,9 +3,9 @@ name: CI
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
```

```suggestion
  # Triggers the workflow on push or pull request events but only for the master and develop branch
```

Is this change correct?

---

I think it is, @sumitramanga - we want the conventional commits enforced for changes into `develop` as well.